### DISCUSSION TITLE
sandbox draggable-list example has a bug

### DISCUSSION THREAD

**Discussion by jky0ung** [pmndrs/react-spring#1655] [Support]
I just find a tiny bug in CodeSandbox example(drabble-list).
It worked well in v8 example, but v9 version doesn't recognize a changed index well.
I think, this happens because height of <div> has been changed by css or other reason.

https://codesandbox.io/s/github/pmndrs/react-spring/tree/master/demo/src/sandboxes/draggable-list

**Comment by joshuaellis** [comment]
This isn't a bug with the library, so I'm going to move this to a discussion. If you'd like to submit a PR to fix the demo, then please feel free to do so! ⭐
